src/dex-ui/store/poolsSlice/poolsSlice.ts

```diff
@@ -188,6 +192,10 @@ const createPoolsSlice: PoolsSlice = (set, get): PoolsStore => {
               successPayload: null,
               errorMessage: "",
             };
+            pools.poolsPageState = {
```

Comment:
Note for the future - we should move **loading**, **routing**, and **transaction state tracking data** into a singular store (`app`) than can be used across all pages.

Reply:
Update: Routing will be handled at the UI layer (hooks) - not in the store. React router should give us everything we need.